package.json

```diff
@@ -91,6 +91,7 @@
   },
   "dependencies": {
     "@redtea/format-axios-error": "^2.0.2",
+    "@workflowwin/iconik-api": "2.2.1",
```

Comment:
We don't need iconik-api in this project.
win.sls has it and should be used for WIN projects as a template